src/auth/urls.py

```diff
@@ -67,72 +62,75 @@ async def login_user(
         # clean old sessions after sending the response
         background_tasks.add_task(crud.task_clean_expired_user_sessions, db_session)
 
-        response = RedirectResponse(next_url)
+        response = RedirectResponse(FRONTEND_ROOT_URL + redirect_path + "#" + redirect_fragment)
         response.set_cookie(
             key="session_id", value=session_id
         )  # this overwrites any past, possibly invalid, session_id
         return response
 
 
-# TODO: deprecate this when possible
 @router.get(
-    "/check",
-    description="Check if the current user is logged in based on session_id from cookies",
+    "/logout",
+    description="Logs out the current user by invalidating the session_id cookie",
 )
```

Comment:
should logout be POST? I think so, since it's about sending a command to the server, not retrieving data. This is also how it was before this PR

Reply:
Weeeeeell... The way I normally think about it is that POST & PATCH should have data that should be POSTed or PATCHed with. GET is the catch-all default otherwise; and you could think about it as GETting whether or not the login or logout was successful. But, that's just how I think about it.

Also, it lets you use your browser to logout with the API route directly, which i do with SFU CAS's API sometimes. You can just go to https://cas.sfu.ca/cas/logout and it'll work, lol.

Anyhoo, I don't really like POSTing without an actual body to be POSTed, but I can change it if you think it's super important

Reply:
My understanding was that POST refers to some kind of mutation occurring to the server, whereas GET requests have no side-effects. Looking into it a bit deeper, this is actually strictly enforced; GET requests must be idempotent! Because of this, some intermediate servers will cache GETs, but not POSTs. I would really suck if the response to `/logout` got cached by a proxy and you're not actually logged out.

see https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET (Cacheable & Idempotent)

Reply:
Then what about the GET login? would that not be cached too? Also I didn't know about this caching, super neat

Reply:
also you have officially changed the way i think about GET & POST requests, me likey :D